docs/usage.md

```diff
@@ -3,51 +3,82 @@
 
 # Usage
 
-The **SOC Overview Sankey** widget helps visualise relationship data as per behavior. For example, viewing the data as per status of related incidents of alerts group by source and severity.
+The **SOC Overview Sankey** widget helps visualize relationship data as per behavior. For example, viewing the data as per status of related incidents of alerts group by source and severity.
 
 You can add this view to a dashboard, or report.
 
-## Features
+## Visualize Relationship Data for a Preset Interval
 
-- Visualize relationship data as per duration. User can select from standard durations provided as part of widget like 7 Days, 1 Day, and 1 Hour. 
+You can select from following options:
 
+- 7 Days
+- 1 Day
+- 1 Hour
+
+    ![](./res/sankey-chart-time-intervals.png)
+<!-- 
 - Configure the widget by selecting source node, target node, relationship node and setting filters to retrieve the required data.
 
-- The links colors will be picked as per picklist items configuration.
+- The links colors will be picked as per picklist items configuration. -->
 
 ## SOC Overview Sankey Widget
 
 1. Edit a *Dashboard*, or a *Report* and select **Add Widget** button.
 
+    ![](./res/sankey-edit-00.png)
+
 2. Select **SOC Overview Sankey** from the list to bring up the **SOC Overview Sankey** widget's customization modal.
 
-3. Specify the title of the visualisation of relationship data as per module in the **Title** field.
+    ![](./res/sankey-edit-01.png)
+
+3. Specify the title of the visualization of relationship data as per module in the **Title** field.
+
+    ![](./res/sankey-edit-02.png)
 
-4. Select the module, whose data to be visualize as per relationship, in the **Resource** field.
+4. Select the module, whose data is to be represented as a Sankey chart as per relationship, in the **Resource** field.
 
-5. Define the filter criteria using which to filter the data retrieved by this widget (Optional).
+    ![](./res/sankey-edit-03.png)
+
+5. Define the filter criteria using which to filter the data retrieved by this widget (Optional), in the **Filter Criteria** field. To know more about using the filter criteria, refer to the [Nested Filter](https://docs.fortinet.com/document/fortisoar/7.4.3/user-guide/207943/dashboards-templates-and-widgets#Nested-Filters) section of the FortiSOAR&trade; user guide.
+
+    ![](./res/sankey-edit-04.png)
 
 6. Select the text field to be grouped for source nodes, in the **Source Nodes** field.
```

Comment:
text field or picklist?

Reply:
fields of type text listed in the picklist